app/types/api/feature.ts

```diff
@@ -9,4 +9,6 @@ export interface Feature {
   alias: string;
   scenarioUsageCount: number;
   tag?: string;
+  splitSelected?: string;
+  splitFeaturesSelected?: string[];
```

Comment:
These properties do not come from the API, so we cannot add them here. I you need additional properties, try extending the type wherever you need it.